src/parser/mod.rs

```diff
@@ -1274,6 +1274,70 @@ impl<'a> Parser<'a> {
         }
     }
 
+    pub fn parse_utility_options(&mut self) -> Result<UtilityOptionList, ParserError> {
+        self.expect_token(&Token::LParen)?;
+        let mut options = vec![];
+        loop {
+            options.push(self.parse_utility_option()?);
+
+            if self.peek_token() != Token::Comma {
+                break;
+            }
+            self.next_token();
+        }
+
+        self.expect_token(&Token::RParen)?;
+
+        Ok(UtilityOptionList { options })
+    }
+
+    fn parse_utility_option(&mut self) -> Result<UtilityOption, ParserError> {
+        let name = self.parse_identifier(false)?;
+
+        let next_token = self.peek_token();
+        if next_token == Token::Comma || next_token == Token::RParen {
+            return Ok(UtilityOption { name, arg: None });
+        }
+        let arg = match next_token.token {
```

Comment:
I think this can be simplified if we used an Expr representation instead of a string (thinking just a call to self.parse_expr() would suffice here as a result)

Reply:
changed. see above.